src/devices/controller/controller.cpp

```diff
@@ -358,9 +400,66 @@ void setup() {
     pinMode(kRightButtons[i], INPUT_PULLUP);
     pinMode(kBottomButtons[i], INPUT_PULLUP);
   }
+
+  if (!digitalRead(kLeftButtons[0])) {
```

Comment:
Nit: Add a TODO here to add it to one of the "function" buttons on the bottom of the controller (and move "0" or the whole symbol into a `const`)

Reply:
Done.